Dockerfile

```diff
@@ -1,56 +1,33 @@
-FROM ubuntu:18.04
+FROM ubuntu:23.04
 
-# Install needed packages
-RUN apt update && apt install -y \
-    apt-transport-https \
-    build-essential \
-    cargo \
-    ca-certificates \
-    clang-10 \
-    curl \
-    chrpath \
-    default-jdk \
-    freeglut3 \
-    freeglut3-dev \
-    g++-8 \
-    gcc-8 \
-    gnupg \
-    gzip \
-    help2man \
-    lcov \
-    libgl1-mesa-dev \
-    libglfw3-dev \
-    libglu1-mesa \
-    libjson-perl \
-    libperlio-gzip-perl \
-    libqt5opengl5-dev \
-    libssl-dev \
-    libx11-dev \
-    libxcursor1 \
-    libxt-dev \
-    openssh-client \
-    pkg-config \
-    python3-dev \
-    qtbase5-dev \
-    software-properties-common \
-    unzip \
-    wget \
-    xvfb
+RUN apt update
```

Comment:
One issue I found earlier is if we update apt in an independent step and you change a package in another step, apt has an outdated package database and there is a good chance it will fail to download the requested package.